.vib/grafana/cypress/cypress/support/commands.js

```diff
@@ -12,6 +12,16 @@ for (const command of ['click']) {
   });
 }
 
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // In Grafana 9.3.0 the console throws a typeError with message
+  //'Cannot read properties of undefined (reading 'eventTrackingNamespace')'.
```

Comment:
Could you add a space just after the slashes to follow the same structure as the other lines, please?